charts/ai-software-templates/chatbot/templates/tekton-repository.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: "pipelinesascode.tekton.dev/v1alpha1"
 kind: Repository
 metadata:
   name: {{ .Release.Name }}-tekton-repository
-  namespace: {{ .Release.Namespace }}
```

Comment:
same why did we remove the namespace question

Reply:
I'm closing in favor of https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/31#discussion_r1923592267

It will be handled similar to what we decide in this conversation